### DISCUSSION TITLE
How do I add bullets (ul) inside a cell in Huxtable?

### DISCUSSION THREAD

**Discussion by TheFirstHonour** [quarto-dev/quarto-cli#11631] [Q&A]
### Description

Hello. I am trying to produce tables in which some cells contain unordered lists. I am trying to output to docx. 

I tried the regular **qmd** tables, and **flextable**, and now I am in **Huxtable**.

The reason I chose to stick with **Huxtable** is because it provides better formatting options for my needs. 

The one thing I am stuck on, which produces the error at the end of this post, is when I try to do bulleted lists inside a cell. Even when trying to use markdown, its failing to produce bulleted lists. 

I tried different methods over the week and honestly I don't know what to do.  

The code that is causing the issue: "information[2, 1] <- "- test 1 \n - test 2 \n - test 3""

Also, 2 small issues: 
1. the font is not being set to **Calibri** in the table. So if any of you can help with that, I would be grateful. 
2. I cannot merge cells in huxtable markdown, is there a workaround to that?

Code: 

````
---
format: 
  docx:
    reference-doc: custom-reference.docx
    number-sections: false
    filters:
      - custom-filters.lua
    tbl-colwidths: auto
---

```{r}
#| echo: false
#| warning: false
#| message: false


library(huxtable)

apply_global_styles <- function(ht) {
  left_border(ht) <- 1
  right_border(ht) <- 1
  bottom_border(ht) <- 1
  top_border(ht) <- 1
  align(ht)[1, ] <- "center"
  font(ht) <- "Calibri"
  font_size(ht) <- 10
  width(ht) <- 1.15
  position(ht) <- "center"
  bold(ht)[, 1] <- TRUE
  background_color(ht)[1, ] <- "#DBE5F1"
  ht
}


```


<!-- Information -->
```{r}
#| echo: false
#| warning: false
#| message: false

library(huxtable)
library(officer)
library(systemfonts)

information <- hux(
    columnone = c("", ""),
    add_colnames = FALSE
)
information[1, 1] <- "test"
information[2, 1] <- "- test 1 \n - test 2 \n - test 3"




# use global styles
information <- apply_global_styles(information)

# bold first column
bold(information)[,1] <- TRUE 
bold(information)[2,] <- TRUE
# shading DBE5F1
background_color(information)[, 1] <- "#DBE5F1"

set_markdown(information)
```

````

And this error happens: 

````
processing file: test.qmd
1/5
2/5 [unnamed-chunk-1]
3/5
4/5 [unnamed-chunk-2]
Error in dplyr::bind_rows(.) : 
  Argument 1 must be a data frame or a named atomic vector.
Calls: .main ... withRestartList -> withOneRestart -> docall -> do.call -> fun



Quitting from lines 41-69 [unnamed-chunk-2] (test.qmd)
Execution halted
````

**Comment by mcanouil** [comment]
This repository is about Quarto CLI not the third parties you are using to generate tables.

For instance the error you showed is an R error that you likely can reproduce in a simple R session.

Regarding your list inside cells, see <https://quarto.org/docs/authoring/tables.html#grid-tables> to write Markdown.
For third parties / libraries, the support should come from them. Quarto CLI does not control what flextable/huxtable/etc. are emitting. Check the documentation or those tools and their repository/authors.

Word document styling comes mostly if not entirely from the reference document. The styles have to be set as documented by Pandoc and Quarto, see https://quarto.org/docs/output-formats/ms-word-templates.html

Or the library you are using needs to emit the proper OpenXML code with the font, etc.

Finally, try to make your code small and reproducible when sharing it. We can't use your code as we don't have your reference document, we don't have your Lua filter, etc. You can share a small Git repository if you absolutely need multiple file to illustrate your issue.
